src/modules/controllers/task.controller.js

```diff
@@ -0,0 +1,64 @@
+const Task = require('../../db/models/task/index');
+
+module.exports.getAllTasks = (req, res) => {
+  Task.find().then(result => {
+    res.send({data: result});
+}).catch(err => {
+    res.send(err);
+  });
+};
+
+module.exports.createNewTask = (req, res) => {
+  res.set('Access-Control-Allow-Origin', '*');
+  const body = req.body;
+
+  if ((body.hasOwnProperty('text') || body.hasOwnProperty('isCheck'))) {
+    const task = new Task({
+      text: body.text,
+      isCheck : body.isCheck
+      });
+
+    task.save().then(result => {
+      res.send(result);
+    }).catch(err => res.send(err));
+  }
+};
+
+module.exports.changeTaskInfo = (req, res) => {
```

Comment:
it is best to divide editing into two "put" requests: one changes the checkbox, the other text